doc/rst/language/spec/expressions.rst

```diff
@@ -856,6 +856,10 @@ The modulus operators are predefined as follows:
 For each of these definitions that return a value, the result is the
 remainder when the first operand is divided by the second operand.
 
+.. note::
+   This does not have the same behavior as :proc:`~AutoMath.mod` when `b` is
+   negative.
+
```

Comment:
This makes more sense to me after the following paragraph and code block; where you define the rule and then list this note, rather than listing the note than define the rule.